src/main/java/frc/robot/RobotContainer.java

```diff
@@ -88,56 +88,31 @@ public RobotContainer() {
    * {@link JoystickButton}.
    */
   private void configureButtonBindings() {
-    new JoystickButton(r_attack3, 2)
-        .whileTrue(new RunCommand(
-            () -> m_robotDrive.setX(),
-            m_robotDrive));
-    
-    m_driverController.y().onTrue(new InstantCommand(m_Corallator::angleDown));
-    m_driverController.a().onTrue(new InstantCommand(m_Corallator::angleUp));
-
-    m_driverController.leftBumper().whileTrue(new StartEndCommand(m_Corallator::outtakeCoral,m_Corallator::stopCoral));
-    m_driverController.rightBumper().whileTrue(new StartEndCommand(m_Corallator::intakeCoral,m_Corallator::stopCoral));
+    r_attack3.button(2).whileTrue(new RunCommand(m_robotDrive::setX));
+    r_attack3.button(7).onTrue(new InstantCommand(m_robotDrive::zeroHeading));
 
+    //upa nd down
+    m_driverController.povDown().onTrue(new InstantCommand(m_Corallator::angleDown));
+    m_driverController.povUp().onTrue(new InstantCommand(m_Corallator::angleUp));
+   
     // intake and outtake
-    JoystickButton unbing = new JoystickButton(r_attack3, 1);
-    unbing.onTrue(new InstantCommand(m_Corallator::outtakeCoral));
 
-    JoystickButton bing = new JoystickButton(r_attack3, 2);
-    bing.onTrue(new InstantCommand(m_Corallator::intakeCoral));
+    m_driverController.leftTrigger().whileTrue(new StartEndCommand(m_Corallator::outtakeCoral,m_Corallator::stopCoral));
+    m_driverController.rightTrigger().whileTrue(new StartEndCommand(m_Corallator::intakeCoral,m_Corallator::stopCoral));
```

Comment:
Probably these two triggers should execute `intakeCoral` and `outtakeCoral`.  Pick another button to do `stopCoral`.